L1Trigger/Phase2L1ParticleFlow/src/ref/pfalgo2hgc_ref.h

```diff
@@ -0,0 +1,17 @@
+#ifndef PFALGO2HGC_REF_H
+#define PFALGO2HGC_REF_H
+
+#include "../firmware/pfalgo2hgc.h"
+#include "pfalgo_common_ref.h"
+
+void pfalgo2hgc_ref_set_debug(int debug);
+
+void pfalgo2hgc_ref(const pfalgo_config &cfg,
+                    const HadCaloObj calo[/*cfg.nCALO*/],
```

Comment:
why not vectors?

Reply:
because it matches the interface of the same functions in the firmware, and those need C-style array and not std::vector